src/shorthands/JImage.jl

```diff
@@ -1,13 +1,21 @@
 function _JImage(pos, img, centering, shapeargs, shape, scaleargs)
-    if shape != false
+    if !isnothing(shape)
         shape(shapeargs...)
     end
     scale(scaleargs)
     placeimage(img, pos, centered = centering)
     return pos
 end
 
-JImage(pos::Point, img, centering = true; shapeargs = (), shape = false, scaleargs = 1) =
+"""
+    JImage(pos::Point, img, centering = true; shapeargs = (), shape = nothing, scaleargs = 1)
+
+Place a given image at a given location as a `Javis` object.
+Images can be cropped to different shapes and scaled to different sizes while being placed.
```

Comment:
make sure to explain the arguments in more detail i.e. based on the docstring I don't really know what `img` is. Is it a filename or an image matrix?

Reply:
Fixed